app/models/user.rb

```diff
@@ -0,0 +1,29 @@
+class User < ActiveRecord::Base
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
```

Comment:
| Given [ERD ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) |
|-|
| ![image](https://github.com/Bestbynature/blogify/assets/93039359/d1f39ccc-876e-4f2b-aa27-dc307de8ba0c) |

- [ ] It would be nice if you make your association according to the given [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) 👍 because in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) you can see that only posts table has author_id but likes and comments table has user_id 🤔 Kindly make this change to follow the project requirements 😊 You can make your associations as below ⬇️

```ruby
  has_many :posts, foreign_key: 'author_id'
  has_many :comments
  has_many :likes
```